modules/console/pages/config/enterprise-license.adoc

```diff
@@ -0,0 +1,74 @@
+= Add a License Key to Redpanda Console
+:description: Learn how to apply or update a license key to Redpanda Console.
+
+To enable xref:get-started:licensing/overview.adoc#console[enterprise features for Redpanda Console], it must have an Enterprise Edition license to load at startup. This guide explains how to configure Redpanda Console to load the license key from its local configuration.
+
+TIP: Redpanda Console can also load the license key from a connected Redpanda cluster. To add a license key to Redpanda, see xref:get-started:licensing/add-license-redpanda/index.adoc[].
```

Comment:
Same comment as on previous pages. Maybe this should be an partial?

Reply:
I could only find this content in one place. Could you link me to the other instance?

Reply:
I just meant it's on multiple pages, but I think I have found them all.